jobs/build/ocp/Jenkinsfile

```diff
@@ -642,11 +661,81 @@ Please direct any questsions to the Continuous Delivery team (#aos-cd-team on IR
                 sh "ose_images.sh --user ocp-build build_container --branch rhaos-${BUILD_VERSION}-rhel-7 --group base --repo https://raw.githubusercontent.com/openshift/aos-cd-jobs/master/build-scripts/repo-conf/aos-unsigned-building.repo"
             }
 
-            buildlib.oit """
+            waitUntil {
+              try {
+                exclude = ""
+                if ( BUILD_EXCLUSIONS != "" ) {
+                  exclude = "-x ${BUILD_EXCLUSIONS} --ignore-missing-base"
+                }
+                buildlib.oit """
 --working-dir ${OIT_WORKING} --group openshift-${BUILD_VERSION}
+${exclude}
 images:build
 --push-to-defaults --repo-type unsigned
 """
+                return true // finish waitUntil
+              }
+              catch ( err ) {
+                record_log = buildlib.parse_record_log( OIT_WORKING )
+                builds = record_log['build']
+                failed_map = [:]
+                for(i = 0; i < builds.size(); i++) {
+                  distgit = build[i]['distgit']
+                  if ( builds[i]['status'] == '-1' ){
+                    task_url = "No Brew task available, see jenkins log."
+                    if ( build[i].containsKey('task_url') ) {
```

Comment:
Why would this ever be missing? In other oit records, I just set the field to "?" so that it will always be there and the pipeline code can stay tight.

Reply:
It won't exist if the brew task is never created: https://github.com/openshift/enterprise-images/blob/2cf0d60b5e66620de8abc8145bc8f483baee8206/oit/packages/distgit.py#L539
Down on L556 is the only place task_url is added to the record. And if the task is never created, it's never added, so it wouldn't exist.

Reply:
I've added: https://github.com/openshift/enterprise-images/commit/78f4ea3c775de2fb3141863731dee56a692ebc38